### ISSUE TITLE
How to reproduce the results of online RL?

### ISSUE THREAD

**Issue by zhxieml** [hij0527/OPAL-PyTorch#2]
Hi,

Thanks for sharing this great implementation of OPAL! I encountered an issue reproducing the results of online RL (i.e., Table 3 of the paper). Wish you can kindly provide some suggestions.

I tried to reproduce the results of SAC+OPAL in Table 3, in the antmaze-medium-sparse-diverse environment. To learn the primitives, I first used D4RL's script to generate antmaze offline dataset, placed it in the `data/` folder, and ran 

```
python train.py
```

Then, I tried to learn an online RL agent with OPAL by running

```
python train_task.py --opal_ckpt results/checkpoints/${env_name}/opal_100.ckpt --task_type online --policy_type sac
```

Unfortunately, it seemed that the agent always achieves zero success rate during the training process (2.5e6 steps) and the policy loss (online_loss/policy) does not drop.

Do you have any idea about how to solve this problem? Reproducing scripts will be very helpful.

Best wishes
